src/jit/codegenxarch.cpp

```diff
@@ -1152,6 +1152,39 @@ void CodeGen::genReturn(GenTreePtr treeNode)
     // Also, there is not much to be gained by materializing it as an explicit node.
     if (compiler->compCurBB == compiler->genReturnBB)
     {
+#if defined(_TARGET_AMD64_)
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
```

Comment:
I *think* what you mean is that the first `#if` is redundant, as it is implied by the `#ifdef`


---
In reply to: [87924333](https://github.com/dotnet/coreclr/pull/7719#discussion_r87924333) [](ancestors = 87924333)